mountpoint-s3/src/fstab.rs

```diff
@@ -0,0 +1,227 @@
+use anyhow::anyhow;
+use clap::Parser;
+use mountpoint_s3_fs::cli::CliArgs;
+use std::env;
+use std::ffi::OsString;
+
+#[derive(Parser, Debug)]
+pub struct FsTabCliArgs {
+    pub bucket_name: String,
+    #[clap(value_name = "DIRECTORY")]
+    pub mount_point: String,
+    #[clap(short = 'o')]
+    pub options: String,
+}
+
+impl FsTabCliArgs {
+    pub fn try_to_cli_args(self) -> anyhow::Result<CliArgs> {
```

Comment:
In Rust, if the method consumes `self`, it usually called as `into`, so maybe: `try_into_cli_args`?

Reply:
I'd not noticed it consumed `self` and I don't think it needs to. I'll change it to taking an `&self`